build-system/tasks/performance/index.js

```diff
@@ -32,13 +33,17 @@ async function performance() {
   await compileScripts(urls);
   await rewriteScriptTags(urls);
   await getMetrics(urls, {headless, runs});
+  runTests();
   printReport(urls);
 }
 
 performance.description = '  Runs web performance test on current branch';
 
 performance.flags = {
   'nobuild': 'Does not compile javascripts before running tests',
+  'threshold':
+    'Percentage above stable metrics that is passable. Number between 0 and 1',
+  'url': 'Page to test. Overrides urls set in config.json',
```

Comment:
Does this mean the override is always a single URL? (Can it take a comma separated list?)

Reply:
Just a URL, having more than 1 makes me think it should be using the config file.